network-framework/channel-artifacts/.gitkeep

```diff
@@ -0,0 +1 @@
+#!/usr/bin/env bash
```

Comment:
Still need to address this.

Reply:
channel-artifacts is an empty folder that will store some files when fabric runs. I don't have a better way to push an empty directory, and putting a gitkeep seems to be the convention. I will delete this file since the folder already exists on gitHub.